examples/Layout/Callout.md

```diff
@@ -0,0 +1,5 @@
+Renders a styled `<p>` element for displaying a piece of text intended to stand out to the user.
+
+```jsx
+<Callout>Callout text</Callout>
+```
```

Comment:
For components whose styling can be changed via props, it's usually good to provide a few examples showing some of the different use cases. So here, we might want to have:

1. `VARIANT.INFO` callout
2. `VARIANT.NEGATIVE` callout (maybe in this case also setting `role: 'alert'`)
3. `VARIANT.POSITIVE` callout

For each, I'd also suggest using text that might plausibly be used in an application, like the "Remember to attend the lab safety training." message we have on the sign-in kiosk. Especially for a block-level component like this, it's good to show how it might look with different amounts of text.